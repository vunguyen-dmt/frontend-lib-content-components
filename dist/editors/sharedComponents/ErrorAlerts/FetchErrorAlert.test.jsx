import React from 'react';
import { shallow } from 'enzyme';
import { FetchErrorAlert } from './FetchErrorAlert';

jest.mock('../../data/redux', () => ({
  selectors: {
    requests: {
      isFailed: jest.fn((state, params) => ({ isFailed: { state, params } })),
    },
  },
}));

describe('FetchErrorAlert', () => {
  describe('Snapshots', () => {
    test('snapshot:  is ErrorAlert with Message error (ErrorAlert)', () => {
      expect(shallow(<FetchErrorAlert isFetchError />)).toMatchSnapshot();
    });
  });
});
