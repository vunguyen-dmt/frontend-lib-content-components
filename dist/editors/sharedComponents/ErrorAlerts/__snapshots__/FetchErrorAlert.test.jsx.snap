// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FetchErrorAlert Snapshots snapshot:  is ErrorAlert with Message error (ErrorAlert) 1`] = `
<ErrorAlert
  dismissError={null}
  hideHeading={false}
  isError={true}
>
  <FormattedMessage />
</ErrorAlert>
`;
